docker-compose.yml

```diff
@@ -0,0 +1,18 @@
+version: '3'
+
+services:
+    web:
+        image: nginx:1.13.7
+        ports:
+            - "8080:80"
+        volumes:
+            - ./config/nginx-default.conf:/etc/nginx/conf.d/default.conf
```

Comment:
configuration is not hardcoded into the image, but mounted from outside. That will be necessary for anything containing credentials, and for anything that changes between environments.